controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentSteps.java

```diff
@@ -39,9 +40,10 @@ public DeploymentSteps(DeploymentInstanceSpec spec, Supplier<SystemName> system)
 
     /** Returns jobs for this, in the order they are declared */
```

Comment:
Not: This javadoc should now say "in the order they should be executed" now.

Reply:
Hehe, yes, I keep changing that doc back and forth :p Will replace the whole class next, to orchestrate instances, anyway.